app/import_functions.go

```diff
@@ -1117,6 +1117,10 @@ func (a *App) ImportDirectChannel(data *DirectChannelImportData, dryRun bool) *m
 		userMap[username] = user.Id
 	}
 
+	if len(userIds) == 1 {
```

Comment:
I considered to introduce this change in the `export_converters` and have JSON lines like this:
```json
{"type":"direct_channel","direct_channel":{"members":["ethervoid","ethervoid"],"favorited_by":null,"header":""}}
{"type":"direct_post","direct_post":{"channel_members":["ethervoid", "ethervoid"],"user":"ethervoid","message":"testing message to myself","create_at":1574429521630,"flagged_by":null,"reactions":null,"replies":null,"attachments":null}}
```
but looks like the JSON is follow the implementation we have instead of something more agnostic, so I decided to implement it in the import workflow instead.